perun-cli/setAttributePolicyCollections

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Text::ASCIITable;
+use Perun::Agent;
+use Perun::Common qw(printMessage);
+use Switch;
+
+sub help {
+	return qq{
+	Sets attribute policy collections
+	--------------------------------
+	Available options:
+	--attributeId       | -a attribute identifier
+	--attributeName     | -A attribute name (including namespace)
+	--policyCollections | -p list of policy collections in the following format:
+
+	 	[ACTION]=[ROLE]:[OBJECT]&&[ROLE]:[OBJECT]||[ACTION]=[ROLE]:[OBJECT]&&[ROLE]:[OBJECT]
+
+	 	ACTION = R (Read), W (Write), R+W (Read + Write)
+	 	ROLE   = one of the available Perun roles.
+	 	OBJECT = None, Group, Vo, Facility, Resource, User, Member
```

Comment:
```suggestion
	 	OBJECT = None, Group, Vo, Facility, Resource, User, Member, SecurityTeam
```

Reply:
Please update also openapi's RoleObject definition as we forgot to update it before. Then it looks all good to me, I like the new formatting of the get script output.